### PR TITLE
Switch to web API

### PR DESCRIPTION
**Pull request by MaxJohansen**
## Changes :balloon:
This offloads the actual image processing logic to a web API hosted on AWS. This is an MVP so the URL is hardcoded and the views need some attention, but it actually works so that's a plus.

## Types of changes

This results in *DRASTIC* changes to the internal API: the Diplomatic.Core library is gone and its classes have been absorbed into the Diplomatic shared codebase.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist :heavy_check_mark:

lol tests, did you even read what I wrote in the last paragraph?

- [x] I have read the coding convention [guidelines](https://github.com/Dualog-students/dualog-students.github.io#company-guidelines) and my code follows these.
- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

After trying several libraries for image manipulation (SkiaSharp, ImageSharp, System.Drawing) I tried implementing the actual diploma generation as a web service written in Python using Flask and Pillow. This was waaay easier.

### CODE REVIEW

File: Diplomatic/ViewModels/ResultViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using System.Linq;
+using Diplomatic.Utils;
+
+
+namespace Diplomatic.ViewModels
+{
+    public class ResultViewModel
+    {
+        public Uri ImageUri { get; }
+        public string Uri => ImageUri.ToString();
```
Comment (jonjohansen) [inline Diplomatic/ViewModels/ResultViewModel.cs]: Residue?

**Review by jonjohansen — DISMISSED** [review]
One smally unusedy boi, but other than that it lgtm!

**Review by vebjornhaugland — DISMISSED** [review]
So beautiful 🏩

File: Diplomatic/ViewModels/ResultViewModel.cs (hunk 1 above)
Comment (IsakSundeSingh) [reply Diplomatic/ViewModels/ResultViewModel.cs]: Such an unused boi

**Review by jonjohansen — APPROVED** [review]

**Pull request merged**